jscomp/bin/whole_compiler.ml

```diff
@@ -81778,7 +81778,19 @@ and
       (Blk_tuple | Blk_array | Blk_variant _ | Blk_record _ | Blk_na | Blk_module _
       |  Blk_constructor (_, 1) (* Sync up with {!Js_dump}*)
       ) 
-      -> expression_desc cxt l f  (Array (el, mutable_flag))
+      ->
+      let string_of_option o = match o with
+      | None -> ""
+      | Some s -> s in
+      let rec process_comments (el:J.expression list) = match el with 
```

Comment:
and this function also to top-level . inner functions are hard to reason about, it may accidentally capture variables